src/pytorch_ood/api.py

```diff
@@ -24,7 +24,13 @@ class ModelNotSetException(ValueError):
     def __init__(self, msg="When using predict(), model must not be None"):
         super(ModelNotSetException, self).__init__(msg)
 
+class AlreadyFittedException(Exception):
```

Comment:
Is this class used anywhere? iirc. the default behavior for detectors that are already fitted is to "refit"